projects/metis/src/app/dataset/datasetlog/datasetlog.component.ts

```diff
@@ -92,10 +92,6 @@ export class DatasetlogComponent extends DataPollingComponent implements OnInit
       () => {
         return of(this.getProcessedCount()).pipe(
           map((val) => {
-            if (isPluginCompleted(this.showPluginLog) && this.subs.length > 0) {
-              this.cleanup();
-              return 0;
-            }
```

Comment:
Are you sure this should be removed? If the task is complete, you should stop polling, right?